src/libraries/Microsoft.PowerFx.Core/Entities/External/CdpDelegationInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.Core.Entities
+{
+    internal class CdpDelegationInfo : TableDelegationInfo
```

Comment:
Can you add a comment about what this is for? Not entirely clear when it's used. Also, not sure it belongs in PFx.Core vs Connectors.